Bug fixes

MrTJP opened the pull request:
* Fixed light halos not rendering properly in non-fabulous modes (Fixes #1957)
* Fixed crash when placing Toggle Latch by slightly altering the lever model (Fixes #1959)

MrTJP committed:
fix: light halos not rendering properly in non-fabulous modes

Floppy012 commented:
@MrTJP Is there anything blocking this MR or do you need some help? Currently dealing with this bug and would be happy if it gets fixed 😄

MrTJP commented:
Will merge and push tonight, have been away for a while :)

dequbed committed:
fix: client crash when placing toggle latch

The renderer in use can't deal with vertices that are not face-aligned
to block orientations and thus crashes on the inclines faces of the lever
model. Changing the lever model to have straight faces with a sliding
lever instead of a gyrating one fixes this crash.

Fixes: #1959
